java/runtime-common/src/main/java/org/ray/core/RayRuntime.java

```diff
@@ -374,16 +374,16 @@ public RayObjects call(UniqueID taskId, Class<?> funcCls, RayFunc lambda, int re
       while (ret.getRight() != GetStatus.SUCCESS) {
         RayLog.core.warn(
             "Task " + taskId + " Object " + objectId.toString() + " get failed, reconstruct ...");
-        localSchedulerProxy.reconstructObject(objectId);
+        localSchedulerProxy.reconstructObject(objectId, false);
 
         // Do another fetch
-        objectStoreProxy.fetch(objectId);
+        objectStoreProxy.fetch(objectId, false);
 
         //Check the result every 5s, but it will return once available.
         ret = objectStoreProxy.get(objectId, params.default_get_check_interval_ms,
             isMetadata);
       }
-      RayLog.core.debug(
+      RayLog.core.warn(
```

Comment:
why change this?

Reply:
I have not changed it. It should be master branch changed it into `warn`.
I'am going to sync this to master.